### PR TITLE
Vectorization Integration

### PR DESCRIPTION
**Pull request by sumitnautiyalopensource**

### CODE REVIEW

File: platform-jobs/samza/mvc-processor-indexer/src/main/config/local.mvc-processor-indexer.properties [hunk 1]
```diff
@@ -74,4 +74,5 @@ definitions.update.window.ms=300000
 # Filter Metadata based on Definition while indexing into ES.
 restrict.metadata.objectTypes=Content,ContentImage
 
-kp.content_service.base_url=__kp_content_service_base_url__
\ No newline at end of file
+kp.content_service.base_url=http://15.2.8.67/content
```
Comment (krgauraw) [inline platform-jobs/samza/mvc-processor-indexer/src/main/config/local.mvc-processor-indexer.properties:77]: please remove this.

File: platform-jobs/samza/mvc-processor-indexer/src/main/java/org/ekstep/mvcjobs/samza/service/util/MVCProcessorESIndexer.java [hunk 2]
```diff
@@ -53,15 +54,36 @@ public void upsertDocument(String uniqueId, Map<String,Object> jsonIndexDocument
...
-		} else {
-			// Update a doc
-			ElasticSearchUtil.updateDocument(CompositeSearchConstants.MVC_SEARCH_INDEX,
-					uniqueId, mapper.writeValueAsString(jsonIndexDocument));
+		String jsonindexasString = mapper.writeValueAsString(jsonIndexDocument);
+		String jsonindexasString2 = "";
+		switch (action) {
+			case "update-es-index" :
+				// Insert a new doc
+				CompletableFuture.runAsync( () -> {
```
Comment (krgauraw) [inline platform-jobs/samza/mvc-processor-indexer/src/main/java/org/ekstep/mvcjobs/samza/service/util/MVCProcessorESIndexer.java:62]: please revert from async calls

**Review by krgauraw — CHANGES_REQUESTED** [review]

File: platform-jobs/samza/mvc-processor-indexer/src/main/config/local.mvc-processor-indexer.properties (hunk 1 above)
Comment (sumitnautiyalopensource) [reply platform-jobs/samza/mvc-processor-indexer/src/main/config/local.mvc-processor-indexer.properties]: change made

File: platform-jobs/samza/mvc-processor-indexer/src/main/java/org/ekstep/mvcjobs/samza/service/util/MVCProcessorESIndexer.java (hunk 2 above)
Comment (sumitnautiyalopensource) [reply platform-jobs/samza/mvc-processor-indexer/src/main/java/org/ekstep/mvcjobs/samza/service/util/MVCProcessorESIndexer.java]: change made

**Pull request closed without merging**